handbook/engineering/engineering-management.md

```diff
@@ -49,20 +49,43 @@ Here are some books that multiple people on the team have read and recommend. If
+For teammates pursuing an Engineering Manager role but who lack the necessary people management experience, we have defined the following transitioning guide that affords the candidate a "manager in training" opportunity to try out being an Engineering Manager for a trial period. The trial period will allow the teammate to determine if they want to commit to it long-term and verify that they would be good in the role.
 
-In some cases, it may not be practical, or headcount planning might not allow for the Interim Manager to make a new hire. In this case, the Interim Manager and the interim manager's manager should agree on success criteria based on the role's requirements.
+### Expressing interest in becoming an Engineering Manager
 
-Once the Interim Manager's first new hire has been at Sourcegraph for 30 days or other agreed success criteria have been met, their manager will formally review the Interim Manager.
+Teammates who have reached the [IC3 level](career-development/framework.md#ic3) and have a track record of good performance at Sourcegraph, having been employed for at least six months, are eligible for a manager in training trial.
+The teammate should make their desire clear to their Engineering Manager so that when a relevant opportunity becomes available, they may be considered.
 
-This process aims to have a determination made by the interim manager's manager about the promotion to a permanent manager role after the agreed success criteria have been satisfied. Generally, we aim to have the interim period not exceed four months before turning into a full-time position.
+Opportunities for a manager in training trial can come about through a current Engineering Manager position becoming vacant, an existing team splitting into two, or an Engineering Manager going on long-term PTO. 
+When an opportunity arises, the teammate's manager can nominate them for a manager in training trial to the Director of Engineering for the relevant org.
+
+There are some considerations to keep in mind to ensure that the teammate is set up for success:
+- The teammate's existing team should be in a position for them not to be involved as an IC for the whole duration of the trial period. Managers in training should not fulfill a dual role.
+- The team to be managed should comply with our [minimum viable team](eng_org.md#minimum-viable-team) criteria.
+- The team to be managed should have a dedicated and established Product Manager assigned to it.
+
+### Manager in training trial period
+
+During the trial period, the manager in training will assume full Engineering Manager responsibility as outlined in the [Engineering Manager responsibilities](roles.md#engineering-manager).
+
+The teammate's manager or another nominated manager will become their sponsoring manager for the duration of the trial period. The sponsoring manager must commit to providing extra support to the manager in training during the trial period, which will last 12 weeks. 
```

Comment:
Does this have to be an EM or can it be a director too (e.g. in the case of the manager going on long-term PTO)?

Reply:
@felixfbecker to clarify are you asking whether a director could also be the sponsoring manager rather than just an EM? If so, then absolutely yes.

Reply:
Yep, that's what I'm asking. If that's the case I'd suggest explicitly saying "manager or director" here to avoid misunderstandings.

Reply:
```suggestion
The teammate's manager, director, or another nominated manager/director will become their sponsoring manager for the duration of the trial period. The sponsoring manager must commit to providing extra support to the manager in training during the trial period, which will last 12 weeks. 
```